esda/losh.py

```diff
@@ -0,0 +1,134 @@
+import numpy as np
+import warnings
+from scipy import sparse
+from scipy import stats
+from sklearn.base import BaseEstimator
+import libpysal as lp
+
+
+class losh(BaseEstimator):
+    """Local spatial heteroscedasticity (LOSH)"""
+
+    def __init__(self, connectivity=None, inference=None):
+        """
+        Initialize a losh estimator
+
+        Arguments
+        ---------
+        connectivity     : scipy.sparse matrix object
+                           the connectivity structure describing the
+                           relationships between observed units.
+        inference        : str
+                           describes type of inference to be used. options are
+                           "chi-square" or "permutation" methods.
+
+        Attributes
+        ----------
+        Hi               : numpy array
+                           Array of LOSH values for each spatial unit.
+        ylag             : numpy array
+                           Spatially lagged y values.
+        yresid           : numpy array
+                           Spatially lagged residual values.
+        VarHi            : numpy array
+                           Variance of Hi.
+        pval             : numpy array
+                           P-values for inference based on either
+                           "chi-square" or "permutation" methods.
+        """
+
+        self.connectivity = connectivity
+        self.inference = inference
+
+    def fit(self, y, a=2):
```

Comment:
`a=2` should be in the init, and again on `X` and `y` as the arguments here.

Reply:
Moved `a=2` to init, and given that losh is univariate changed `y` to `x` in `fit(...)`, can you confirm on https://github.com/pysal/esda/pull/139/commits/3b47da838127a6e9677158ac9d11a2b321a8bef9?